libs/utils/android/benchmark.py

```diff
@@ -116,7 +116,7 @@ def _parseCommandLine(self):
 
         # Test customization
         parser.add_argument('--results-dir', type=str,
-                default=None,
+                default=self.__class__.__name__,
```

Comment:
Yeah, that could be a good idea. I do think having the date+time is a good idea rather than overwriting data when repeating tests. I suppose am entitled millenial but disk space is cheap :stuck_out_tongue: